torchaudio/transforms/_transforms.py

```diff
@@ -1385,6 +1385,13 @@ class Vad(torch.nn.Module):
         lp_lifter_freq (float, optional) "Brick-wall" frequency of low-pass lifter used
             in the detector algorithm. (Default: 2000.0)
 
+    Example
+        >>> waveform, sample_rate = torchaudio.load('test.wav', normalize=True)
+        >>> waveform_reversed, sample_rate = apply_effects_tensor(waveform, sample_rate, [['reverse']])
+        >>> transform = transforms.Vad(sample_rate=sample_rate, trigger_level=7.5)
+        >>> waveform_reversed_front_trim = transform(waveform_reversed)
+        >>> waveform_end_trim = apply_effects_tensor(waveform_reversed_front_trim, sample_rate, [['reverse']])
```

Comment:
```suggestion
        >>> waveform_end_trim, sample_rate = apply_effects_tensor(waveform_reversed_front_trim, sample_rate, [['reverse']])
```

Reply:
Committing, thanks! 

As a side note, I was thinking of a potential enhancement to automate the "reversing process" in this example by adding a boolean parameter to the interface of `Vad` such as `trim_front=True`; perhaps I'll open a new issue for this enhancement idea to solicit feedback from you and other maintainers (so as not to crowd this PR as well!)